patch_parser.py

```diff
@@ -316,6 +316,8 @@ def diff_reply(parent: Message, child: Message) -> List[Comment]:
 def filter_patches_and_cover_letter_replies(email_thread: Message) -> (List[Message], List[Message]):
     patches = []
     cover_letter_replies = []
+    if (not email_thread.in_reply_to and re.match(r'\[.+ 1/\d+\] .+', email_thread.subject)):
```

Comment:
would using `parse_set_index(email_thread)` be cleaner here?
We'd just check that the first value is 1.